validator/project_validator.go

```diff
@@ -259,13 +264,13 @@ func CheckYamlStrict(yamlBytes []byte) ValidationErrors {
 // verify that the project configuration semantics and configuration syntax is valid
 func CheckProjectConfigurationIsValid(project *model.Project, pref *model.ProjectRef) error {
 	catcher := grip.NewBasicCatcher()
-	syntaxErrs := CheckProjectSyntax(project, false)
+	syntaxErrs := CheckProjectErrors(project, false)
 	if len(syntaxErrs) != 0 {
 		if errs := syntaxErrs.AtLevel(Error); len(errs) != 0 {
 			catcher.Errorf("project contains syntax errors: %s", ValidationErrorsToString(errs))
 		}
 	}
-	semanticErrs := CheckProjectSemantics(project)
+	semanticErrs := CheckProjectWarnings(project)
```

Comment:
Are we certain now that all of the errors returned here are at the warning level? In which case I'm not sure we should be checking this at all here, since we only add errors to the catcher.

Reply:
yes it returns all warnings